instrumentation/armeria/armeria-1.3/library/src/main/java/io/opentelemetry/instrumentation/armeria/v1_3/ArmeriaTelemetryBuilder.java

```diff
@@ -114,14 +76,14 @@ public ArmeriaTelemetryBuilder addAttributeExtractor(
   @CanIgnoreReturnValue
   public ArmeriaTelemetryBuilder addClientAttributeExtractor(
       AttributesExtractor<? super ClientRequestContext, ? super RequestLog> attributesExtractor) {
-    additionalClientExtractors.add(attributesExtractor);
+    clientBuilder.addAttributeExtractor(attributesExtractor);
```

Comment:
here we split client/server AttributesExtractor, it seems to me like we should split them all (e.g. client/server SpansStatusExtractor)

which then leads to the question of should we split these common client/server builders in two classes (e.g. at least this and grpc, I don't remember if there are any others)

Reply:
this refactoring is not changing the external API so far - are you suggesting we should do that?

Reply:
> this refactoring is not changing the external API so far

ya, good point, let's stick to the way you have it in this PR at least